ref/config/serverConfiguration.adoc

```diff
@@ -24,171 +18,134 @@ following example:
-supported by a server depend on which features the server is configured so unknown
-config is simply ignored.
+The most important and only required configuration file is the `server.xml` file. The `server.xml` file must be well-formed XML and the root element must be `server`. The exact elements that are supported by a server depend on which features are configured, and any unknown config is ignored.
 
-{projectName} uses a principle of configuration by exception, allowing for very
-succinct config files. The runtime environment operates from a set of
-built-in config default settings. You only specify config that overrides
+{projectName} uses a principle of configuration by exception, which allows for succinct config files. The runtime environment operates from a set of built-in config default settings. You only specify config that overrides
 those default settings.
 
-Server config files are loaded in the following order:
+Server config files are read from the following locations in order:
 
-* `${server.config.dir}/configDropins/defaults/`
-* `${server.config.dir}/server.xml`
-* `${server.config.dir}/configDropins/overrides/`
+. `${server.config.dir}/configDropins/defaults/`
+. `${server.config.dir}/server.xml`
+. `${server.config.dir}/configDropins/overrides/`
 
-The `${server.config.dir}/server.xml` file must be present but the other files
-are optional. This allows a very flexible way to compose config by simply
-dropping server-formatted XML files into directories. Within the two
-`configDropins` directories the files are read in alphabetical order.
+The `${server.config.dir}/server.xml` file must be present, but the other files are optional. You can flexibly compose config by dropping server-formatted XML files into directories. Files are read in alphabetical order in each of the two `configDropins` directories.
 
-=== Variable substitution
-You can parameterize server config using variables. When resolving
-variable names the following sources are consulted in increasing order of
-precedence:
+== Variable substitution
+You can use variables to parameterize the server configuration. To resolve variable names, the following sources are consulted:
 
-* `server.xml` default variable values
-* environment variables
+* `server.xml` file default variable values
+* Environment variables
 * `bootstrap.properties`
 * Java system properties
-* `server.xml` config
+* `server.xml` configuration
 
-Variables are referenced using `${variableName}` syntax. In server config,
-specify variables using the `variable` element:
+In this list, each source takes precedence over the previous source, so that the `server.xml` configuration takes the highest precedence and the  `server.xml` file default variable values take the lowest precedence. Variables are referenced by using the `${variableName}` syntax. You can specify variables in the server configuration by using the  `variable` element, as shown in the following example:
```

Comment:
If the prior list were numbered rather than bulleted then the first added sentence would not be necessary.

Reply:
Given there is an example following I don't know why we need to spell out the variable element in the preceding sentence.

Reply:
Updated both of these. See if the list still reads properly to you with the updates.